app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -70,7 +65,7 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
 
     private val mediaSessionHolder get() = activity as MediaSessionHolder? // null when not attached.
 
-    val isUrlEqualToHomepage: Boolean get() = session.url == APP_URL_HOME
+    private val isUrlEqualToHomepage: Boolean get() = session.url == AppConstants.APP_URL_HOME
```

Comment:
This function is redefined elsewhere: can we define it somewhere generic and reduce duplication?

Reply:
The other call site with this logic performs the check on a session, so the amount of code sharing is really minimal.  It would also change `session.url == URLs.APP_URL_HOME` to `URLs.isHome(session.url)` (or similar), which I don't think is more readable.